src/spikeinterface/extractors/tests/test_nwbextractors.py

```diff
@@ -280,11 +279,96 @@ def test_sorting_extraction_of_ragged_arrays(tmp_path):
     assert "uniform_ragged_array" in added_properties
     assert "a_property" in added_properties
 
-    spike_train1 = sorting_extractor.get_unit_spike_train(unit_id="a", return_times=True)
-    np.testing.assert_allclose(spike_train1, spike_times1)
+    extracted_spike_times_a = sorting_extractor.get_unit_spike_train(unit_id="a", return_times=True)
+    np.testing.assert_allclose(extracted_spike_times_a, spike_times_a)
+
+    extracted_spike_times_b = sorting_extractor.get_unit_spike_train(unit_id="b", return_times=True)
+    np.testing.assert_allclose(extracted_spike_times_b, spike_times_b)
+
+
+def test_sorting_extraction_start_time(tmp_path):
+    nwbfile = mock_NWBFile()
+
+    # Add the spikes
+
+    t_start = 10
+    sampling_frequency = 1.0
```

Comment:
```suggestion
    sampling_frequency = 100.0
```
To make it more challenging!

Reply:
All right.